web/src/components/ResultCardList/ResultCardList.tsx

```diff
@@ -0,0 +1,65 @@
+import {useCallback} from 'react';
```

Comment:
Main controller component for the result cards

Reply:
nice